### ISSUE TITLE
Reviewing Sequence Diagrams

### ISSUE THREAD

**Issue by karalleyna** [bounswe/bounswe2020group2#54]
After #53 , I added some new methods to our class diagram. Then, I reviewed your sequence diagrams and I realize that I can add them to yours. Could I recreate your sequence diagrams considering the last version of our class diagram if it is not problem for you?

**Comment by furkannane** [comment]
There is a new issue for revision of Sequence Diagrams.
